pyccel/codegen/printing/cucode.py

```diff
@@ -62,13 +62,63 @@ def _print_Module(self, expr):
                 local_imports += self._print(imp)
 
         imports = f'{c_headers_imports}\
-                    extern "C"{{\n\
+                   \n\
                     {local_imports}\
-                    }}'
+                    '
 
         code = f'{imports}\n\
                  {global_variables}\n\
                  {body}\n'
 
         self.exit_scope()
         return code
+    def _print_ModuleHeader(self, expr):
+        self.set_scope(expr.module.scope)
+        self._in_header = True
+        name = expr.module.name
+        if isinstance(name, AsName):
+            name = name.name
+        # TODO: Add interfaces
```

Comment:
You can raise an error here if you don't have anything to test with

Reply:
I don't know what you mean ??

Reply:
There is an example below that you have already merged. You now raise a warning if a module should wrap an Interface.